modules/system/assets/ui/docs/input-hotkey.md

```diff
@@ -18,4 +18,16 @@ Allows keyboard shortcuts (hotkeys) to be bound to an element's click event.
         Press "Shift + R" on your keyboard
     </button>
 
+## Javascript API
 
+If you use a selector other than a button or a link, you will need to add the `hotkeyVisible` property to the hotkey config.
+
+```html
```

Comment:
The documentation doesn't utilize language hinting or backticks for code syntax highlighting, indent code examples by four spaces and remove the backticks and then we can merge this. Thanks for your work so far!